src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -69,6 +69,20 @@ public static class ConfigurationKeys
         /// <seealso cref="TracerSettings.AgentUri"/>
         public const string AgentPort = "DD_TRACE_AGENT_PORT";
 
+        /// <summary>
+        /// Configuration key for windows named pipe where the Tracer can send traces.
+        /// Default value is null.
+        /// </summary>
+        /// <seealso cref="TracerSettings.ApmWindowsPipeName"/>
+        public const string ApmWindowsPipeName = "DD_APM_WINDOWS_PIPE_NAME";
```

Comment:
APM => Trace
Windows pipe => Named pipe

```suggestion
        /// Configuration key for the name of the named pipe where the Tracer can send traces.
        /// Default value is <c>null</c>.
        /// </summary>
        /// <seealso cref="TracerSettings.TracesNamedPipeName"/>
        public const string TracesNamedPipeName = "DD_TRACE_NAMED_PIPE_NAME";
```

Reply:
This matches the environment variable in the agent, I'd like to leave it in order to be consistent. Or we could see if @gbbr is interested in changing it trace agent side?

Reply:
`DD_TRACE` is the standard prefix for tracer settings and `DD_APM` is used for Agent settings. I think consistency with other tracer settings (and other tracers) is more important than consistency with the Agent.